### ISSUE TITLE
default-props-match-prop-types doesn't detect properties in quotes

### ISSUE THREAD

**Issue by zech** [yannickcr/eslint-plugin-react#1908]
Hello,

same as in issue [#1201](https://github.com/yannickcr/eslint-plugin-react/issues/1201) last year the above mentioned rule doesn't detect quoted properties. The error for the second and third property is `defaultProp "undefined" defined for isRequired propType. (react/default-props-match-prop-types)`

```javascript
SomeComponent.propTypes = {
    "firstProperty": PropTypes.string.isRequired,
    "secondProperty": PropTypes.bool,
    "thirdProperty": PropTypes.func
};

SomeComponent.defaultProps = {
    "secondProperty": false,
    "thirdProperty": () => undefined
};
```
As before removing the quotes solves the error. But I don't think this should be consistent and a non issue.

**Comment by alexzherdev** [comment]
This one will get automatically fixed when this rules uses the common detection logic from #1911.
